### ISSUE TITLE
Page swiping is not fluid when images  are more than five

### ISSUE THREAD

**Issue by paolospag** [ascoders/react-native-image-viewer#175]
Hi!
For my own project I built an image gallery composed by many small touchable bricks. When users touch on brick, the image is expanded in modal/lightbox.

In order to show zoomable image gallery, I'm using this module because it is exactly what I need.
It works fine when images are less than five (or little more), but when I add so many images, pages swipe seems to lose fluidity.

I reproduce a similar scenario in this snack:
https://snack.expo.io/@wdtester/gallery-with-lightbox

Any idea about this issue?

Thank you.

**Comment by ascoders** [comment]
Hi, `2.2.0` optimizes the performance of showing picture, display only the picture of the current window and another two pictures before and after.

**Comment by d41z** [comment]
@paolospag did you solve it?

**Comment by paolospag** [comment]
@d41z sorry for my late response!

After upgrading to 2.2.0 version the performance seems to have improved, but this have been possibile only when I built APK with no dev mode, instead with dev mode the problem persists.

In general, I noticed that the image release on swiping is not completely "fluid", I would prefer it to be softer and slower (like Facebook image swiper).

However, this is the best module I found and then @ascoders are doing a great work!
Thank you
